Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.61 AS chef
+WORKDIR app
+
+
+FROM chef AS planner
+COPY src src/
+COPY Cargo.* .
```

Comment:
What version of docker are you using? I get 
```
 ---> b8e5c77041e9
Step 5/18 : COPY Cargo.* .
When using COPY with more than one source file, the destination must be a directory and end with a /
```

Seems like specifying `./` as the destination would be backward compatible anyway. This line appears one in `planner` and once in `builder`.